pkg/x/slices/slices.go

```diff
@@ -0,0 +1,18 @@
+package slices
+
+func Merge[T any](slices ...[]T) []T {
+	if len(slices) == 0 {
+		return []T{}
+	}
+
+	var res []T
+	for _, s := range slices {
+		res = append(res, s...)
+	}
+
+	if len(res) == 0 {
+		return []T{}
+	}
+
+	return res
+}
```

Comment:
do we need this? Can we just use a simple `append`?

Reply:
@simar7 Removed this function.